taxonomies/ingredients.txt

```diff
@@ -31389,7 +31401,7 @@ wikidata:en:Q9280616
 wikipedia:en:https://en.wikipedia.org/wiki/Muscovado
 
 # nova:en:invert-sugar
-<en:sugar
+<en:monosaccaride
```

Comment:
All the instances of en:monosaccaride without a h should be changed to en:monosaccharide